source/index.rst

```diff
@@ -6,10 +6,13 @@
 Mixxx |version| User Manual
 ***************************
 
-Mixxx is Free [#f1]_ DJ software for Windows, Mac OS X, and Linux. **Mixxx gives
-you everything you need to become a professional DJ** (except talent and hard
-work). Using Mixxx, you can rock the party with your vinyl turntables, your
-:term:`MIDI`/:term:`HID` controllers, or even just your keyboard.
+Mixxx is free [#f1]_ DJ software for Windows, Mac OS X and Linux. It's
+easy to use, so you will start mixing your favorite songs in no time,
+even if you've never DJed before. Mixxx is also
+powerful, so experienced DJs will find all the advanced features they
+would expect. You can perform with Mixxx using :term:`MIDI` and :term:`HID`
+controllers, CD players, vinyl turntables or even just your
+computer's keyboard and mouse.
```

Comment:
> Mixxx offers all the features an experienced DJ would expect like effects, Vinyl control, Beatloop rolls, automatic Key detection and many more.

I changed the lines because this makes it sound like these features can only be used when you are experienced. While we usually try that everything is easily usable even for beginners. It should bring across the point that the features are suitable for beginners and experienced users.

> I'd keep the "even if you've never DJed before".

I wouldn't keep the 'even if you've never DJed before'. Just playing a track can be done by any program nothing special. The extra features like effects, loops, master-sync, multiple decks are the cool things that allow one to DJ and which differentiate mixxx from a normal music-player like banshee/clementine.

In the end it is a matter of personal taste.